lib/project_types/extension/tasks/run_extension_command.rb

```diff
@@ -47,6 +49,8 @@ def load_server_config
         Tasks::LoadServerConfig.call(
```

Comment:
This method signature reads weird now. It's not obvious why loading the server config requires the resource and tunnel URL. Maybe we should rename this to something like

* `PrepareServerConfig`
* `AssembleServerConfig`
* `MergeServerConfig`

and subsequently rename `file_name` to `source_file`.

Consider this a nice to have not a mandatory refactor.